app/models/host.rb

```diff
@@ -370,7 +334,7 @@ def reset
   end
 
   def start
-    if validate_start[:available] && power_state == 'standby' && respond_to?(:vim_power_up_from_standby)
+    if supports?(:start)
```

Comment:
NOTE all of these checked validates again so I don't want to remove that yet in case something is depending on this
Next I think we should update these to use the standard method_queue, method, raw_method and supports?(:method)

Reply:
very nice improvement here.

is the thought to move the `validate_ipmi` calls into `supports {:start,:stop,:restart}`

Reply:
also, openstack overrides `validate_{start,stop}`

Reply:
> is the thought to move the validate_ipmi calls into supports {:start,:stop,:restart}

Yes exactly, e.g. https://github.com/ManageIQ/manageiq/pull/21847/files#diff-9daa2c365cb8a7cb177648fe53854dae6588080e089b01333768fcb5da8ff4aeR201

> also, openstack overrides validate_{start,stop}

Thanks, put a PR in to openstack as well